src/MessageDispatch.KurrentDB/KurrentDbSubscriber.cs

```diff
@@ -47,16 +51,22 @@ private KurrentDbSubscriber(
         IDispatcher<ResolvedEvent> dispatcher,
         string streamName,
         ILogger logger,
-        ulong? startingPosition)
-        => Init(kurrentDbClient, dispatcher, streamName, logger, startingPosition);
-
+        ulong? startingPosition)
+    {
+        SetupCallbackTimer();

```

Comment:
I think the callback timer would be better placed when we start the subscription as we don't need it until then presumably?